app/views/support/locations/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render PageTitle::View.new(title: "support.providers.locations.new") %>
+
```

Comment:
can we have backlink to provider location index page

Reply:
So far all the forms for editing nested resources in support do not have an explicit bank link. There's a cancel link to return back which does the same job. I think if we add it, we'll need to add it to the others too. We can do it in another ticket